src/ui/atoms/listItem/listItem.scss

```diff
@@ -1,40 +1,84 @@
 @import '../../styles/themes.scss';
+@import '../../styles/animations.scss';
 
 .okp4-listitem-main {
-  display: grid;
-  column-gap: 10px;
-  padding: 10px;
-  align-items: center;
-  grid-template-columns: minmax(30px, 8%) auto auto;
-  grid-template-rows: repeat(2, auto);
-  grid-template-areas:
-    'leftElement title rightElement'
-    'leftElement description rightElement';
-
   @include with-theme {
     background-color: themed(inverted-primary-background);
+    display: grid;
     border-radius: 10px;
-  }
+    gap: 10px;
+    padding: 10px;
+    overflow: auto;
 
-  .okp4-listitem-title {
-    grid-area: title;
-    justify-content: start;
-    margin: 3px;
-  }
+    &.clickable:hover {
+      cursor: pointer;
+      box-shadow: 0 2px 4px themed(inverted-field);
+      animation: 0.2s ease-out hover-animation;
+      transform: translateY(-3px);
+    }
 
-  .okp4-listitem-description {
-    grid-area: description;
-    justify-content: start;
-    margin: 3px;
-  }
+    .okp4-listitem-title {
+      grid-area: title;
+    }
+
+    .okp4-listitem-description {
+      grid-area: description;
+    }
+
+    .okp4-listitem-first-element {
+      grid-area: firstElement;
+    }
+
+    .okp4-listitem-last-element {
+      grid-area: lastElement;
+    }
 
-  .okp4-listitem-left-element {
-    grid-area: leftElement;
-    margin: 5px auto 5px 5px;
+    .okp4-listitem-first-element > img,
```

Comment:
I'm not sure why you are targeting an `img` because I don't see any in the jsx 🤔